mindsdb/integrations/handlers/dropbox_handler/__about__.py

```diff
@@ -0,0 +1,9 @@
+__title__ = "MindsDB Dropbox handler"
+__package_name__ = "mindsdb_dropbox_handler"
+__version__ = "0.0.1"
+__description__ = "MindsDB handler for Dropbox"
+__author__ = "Ton Hoang Nguyen (Bill)"
+__github__ = "https://github.com/mindsdb/mindsdb"
+__pypi__ = "https://pypi.org/project/mindsdb/"
+__license__ = "GPL-3.0"
```

Comment:
This should be MIT

Reply:
@ZoranPandovski Alright, I will change that :)

Just a note in case you are unaware: in the documentation here: https://docs.mindsdb.com/contribute/app-handlers, the GitHub Handler's license is GPL 3.0. However, when I checked the GitHub Handler in the mindsdb repository, it was MIT.

### The `__about__.py `example in the docs on mindsdb website:

<img width="605" alt="Screenshot 2024-10-28 at 20 47 38" src="https://github.com/user-attachments/assets/c21b9fae-c5b1-4316-92b8-a84893eec213">


### The `__about__.py` in the GitHub handler in the mindsdb repository (main branch):

<img width="834" alt="Screenshot 2024-10-28 at 20 50 09" src="https://github.com/user-attachments/assets/d676bbbf-551f-4fb1-9b96-7db296e6bda2">